pkg/api/admin/openshiftcluster.go

```diff
@@ -119,13 +119,23 @@ const (
 	SoftwareDefinedNetworkOpenShiftSDN  SoftwareDefinedNetwork = "OpenShiftSDN"
 )
 
+// MTUSize represents the MTU size of a cluster
```

Comment:
Risking sounding dumb here, but I've read this PR, the card, and searched on Google, but I'm still not 100% sure what "MTU" stands for in this context - is it Maximum Transmission Unit? Either way, can we update this comment to spell out the acronym?